roles/users/tasks/main.yml

```diff
@@ -50,7 +50,7 @@
   changed_when: false
   sudo: no
   register: admin_user_status
-  when: not sshd_permit_root_login
+  when: not sshd_permit_root_login and inventory_hostname != '127.0.0.1'
```

Comment:
Consider an Ansible inventory like this:
```
[web]
nickname ansible_ssh_host=127.0.0.1
``` 
The variable `inventory_hostname` would be `nickname` and wouldn't be captured by your addition to the `when:` conditional. You could check `ansible_ssh_host` first, and fallback to `inventory_hostname`. [Credit here](https://github.com/louim/bedrock-ansible/commit/4b4ff3365b254eaa941c37febf3fbc673f8b5c51#diff-db0d022f251967eca56175a8e2023698R3).